### PR TITLE
Pass down lifecycle stage to filter experiments

### PR DESCRIPTION
**Pull request by eliasbenussi**

### CODE REVIEW

File: mlflow_faculty/mlflow_converters.py [hunk 1]
```diff
@@ -128,3 +130,13 @@ def mlflow_timestamp_to_datetime_milliseconds(mlflow_timestamp):
...
+
+
+def mlflow_viewtype_to_faculty_lifecycle_stage(mlflow_view_type):
+    if mlflow_view_type == ViewType.ACTIVE_ONLY:
+        return FacultyLifecycleStage.ACTIVE
+    elif mlflow_view_type == ViewType.DELETED_ONLY:
+        return FacultyLifecycleStage.DELETED
+    elif mlflow_view_type == ViewType.ALL:
+        return None
+    raise ValueError("Unexpected view_type: {}".format(mlflow_view_type))
```
Comment (pbugnion) [inline mlflow_faculty/mlflow_converters.py]: I feel like we would normally implement this as a dictionary?

**Comment by pbugnion** [comment]
I made one comment inline but modulo that, lgtm

File: mlflow_faculty/mlflow_converters.py (hunk 1 above)
Comment (eliasbenussi) [reply mlflow_faculty/mlflow_converters.py]: Makes sense

File: mlflow_faculty/mlflow_converters.py [hunk 2]
```diff
@@ -128,3 +130,12 @@ def mlflow_timestamp_to_datetime_milliseconds(mlflow_timestamp):
...
 def mlflow_timestamp_to_datetime_seconds(mlflow_timestamp):
     return datetime.fromtimestamp(mlflow_timestamp, tz=UTC)
+
+
+def mlflow_viewtype_to_faculty_lifecycle_stage(mlflow_view_type):
+    conversions = {
+        ViewType.ACTIVE_ONLY: FacultyLifecycleStage.ACTIVE,
+        ViewType.DELETED_ONLY: FacultyLifecycleStage.DELETED,
+        ViewType.ALL: None,
+    }
```
Comment (pbugnion) [inline mlflow_faculty/mlflow_converters.py]: suggest moving conversions to a module level `LIFECYCLE_STAGE_CONVERSION` as discussed

**Pull request merged**